README.md

```diff
@@ -243,6 +243,8 @@ const openAds = OpenAds.init({config: {
 }})
 ```
 
+Navigating in a browser, adding **openads_debug** to the URL's query string, will enable the debug mode automatically
```

Comment:
I would use this sentence: 

"While navigating in a browser, adding **openads_debug** keyword to the URL query string will enable the debug mode automatically."

Maybe an example would be helpful.

Reply:
Done
![](https://media.giphy.com/media/l3vR0ymO2VLhI0XxS/giphy.gif)